nixos/modules/system/boot/networkd.nix

```
@@ -96,6 +98,8 @@ let
         (assertRange "CombinedChannels" 1 4294967295)
         (assertInt "RxBufferSize")
         (assertInt "TxBufferSize")
+        (assertInt "TransmitQueues")
+        (assertInt "TransmitQueueLength")
```

```suggestion
        (assertRange "ReceiveQueues" 1 4096)
        (assertRange "TransmitQueues" 1 4096)
        (assertRange "TransmitQueueLength" 1 4294967294)
```

https://www.freedesktop.org/software/systemd/man/systemd.link.html#TransmitQueues=